cfme/tests/automate/test_service_dialog.py

```diff
@@ -482,3 +485,129 @@ def test_automate_methods_from_dynamic_dialog_should_run_as_per_designed(
         for ele_name in [ele_label, "label2", "label3"]:
             # Checking if automate method is not triggered after updating values of dialog widgets
             view.fields(ele_name).input.fill(fauxfactory.gen_alphanumeric())
+
+
+@pytest.fixture(scope="module")
+def custom_button(appliance):
+    """This fixture creates custom button for user object"""
+    def button(dialog):
+        collection = appliance.collections.button_groups
+
+        # Create custom button group
+        button_grp = collection.create(
+            text=fauxfactory.gen_alphanumeric(),
+            hover=fauxfactory.gen_alphanumeric(),
+            type=getattr(collection, "USER"),
+        )
+
+        # Create custom button
+        button = button_grp.buttons.create(
+            text=fauxfactory.gen_alphanumeric(),
+            hover=fauxfactory.gen_alphanumeric(),
+            dialog=dialog,
+            system="Request",
+            request="InspectMe",
+        )
+        return button_grp, button
+
+    return button
+
+
+def navigation_to_view(custom_button_group, button):
+    """This function helps to navigate on view comes after clicking on custom button"""
```

Comment:
"""This function helps to navigate to view that comes after clicking on custom button"""